src/Microsoft.DotNet.ProjectModel/Compilation/LibraryExporter.cs

```diff
@@ -154,22 +155,26 @@ private LibraryExport GetExport(LibraryDescription library)
             }
         }
 
-        private LibraryExport ExportPackage(PackageDescription package)
+        private LibraryExport ExportPackage(TargetLibraryWithAssets library)
         {
-            var builder = LibraryExportBuilder.Create(package);
-            builder.AddNativeLibraryGroup(new LibraryAssetGroup(PopulateAssets(package, package.NativeLibraries)));
-            builder.AddRuntimeAssemblyGroup(new LibraryAssetGroup(PopulateAssets(package, package.RuntimeAssemblies)));
-            builder.WithCompilationAssemblies(PopulateAssets(package, package.CompileTimeAssemblies));
-            builder.WithSourceReferences(GetSharedSources(package));
-            builder.WithAnalyzerReference(GetAnalyzerReferences(package));
-
-            if (package.ContentFiles.Any())
+            var builder = LibraryExportBuilder.Create(library);
+            builder.AddNativeLibraryGroup(new LibraryAssetGroup(PopulateAssets(library, library.NativeLibraries)));
+            builder.AddRuntimeAssemblyGroup(new LibraryAssetGroup(PopulateAssets(library, library.RuntimeAssemblies)));
+            builder.WithCompilationAssemblies(PopulateAssets(library, library.CompileTimeAssemblies));
+
+            if (library.Identity.Type.Equals(LibraryType.Package))
```

Comment:
All tests that request LibraryExports from nuget packages exercise the branch getting taken.

The branch getting not taken can't be exercised until we decide on #2123